pkg/config/user_config.go

```diff
@@ -502,7 +502,7 @@ func GetDefaultConfig() *UserConfig {
 				OpenStatusFilter:         "<c-b>",
 			},
 			Branches: KeybindingBranchesConfig{
-				CopyPullRequestURL:     "<c-y>",
+				CopyToClipboardMenu:    "<c-y>",
```

Comment:
Could we replace this `<c-y>` with just `y`, like in the `Commits` panel?

Reply:
When I had it as just y it would select the menu, and then just fall through to selecting the next option as well without letting us pick from the menu. Couldn't find why that was the case, maybe you have more insight?

Reply:
Could you check if the code is the same as for copying a commit attribute? It's in the `Commits` pane, also under `y`?

Reply:
Definitely can, will be working on this today

Reply:
Looking into the commit pane impl, the y command for that is also just falling through to the first option. the menu never opens and it just selects the top option. the y key is also a universal binding for `confirmAlt1` so that must be taking precedence. should i switch both of them to c-y? or find another way around it?

Reply:
it does not, it just falls through and selects the first option on master as well for me

Reply:
Uhhh... happy debugging? Try it in a docker container, or with a clean config, and in an another `terminal emulator`.

Reply:
yuuppp, working on figuring it out now. very odd that it's not behaving for me.

Reply:
So experimenting with this, I've tried on two computers, in 3 different shells, and on my branch and in the current brew installation of the app and y is consistently falling through to the first option for me. If it's working for you still then I've got no idea what's going on but I feel like the universal keybinding is taking precedence potentially 🤷‍♂️ But I'm not sure where else to go from here other than maybe mapping them both to c-y, or having them on y and trusting this is an isolated issue for my machines? lol

Reply:
> rusting this is an isolated issue for my machines? lol

Something to be solved, then.

Have you tried:
- different keyboard layout (US seems to work for me)
- in `docker`
- different terminal emulator

Do the integration tests work okay? Can you replay them through the `lazyintegration` replay thingy?

Reply:
I've not tried a different layout, but y works for other commands like c-y so i don't think it's a layout issue but I'll see. I tried in stock MacOS terminal, fish, nushell, kitty, iterm, warp and alacritty (when i said shell above i meant terminal emulator, my bad) both on my desktop and a laptop that was wiped clean about 4 days ago so is basically stock, all on master branch, my personal branches, and a fresh brew install. I haven't tried docker yet, but could give it a shot tomorrow. I ran everything inside of lazyintegration and all of them passed fine. I'll give docker a shot and see, but I'd be lying if I said I wasn't doubtful